release/models/bgp/openconfig-bgp-policy.yang

```diff
@@ -896,9 +916,18 @@ module openconfig-bgp-policy {
     leaf asn {
       type oc-inet:as-number;
       description
-        "The AS number to prepend to the AS path. If this leaf is
-        not specified and repeat-n is set, then the local AS
-        number will be used for prepending.";
+        "The AS number to prepend to the AS path. If neither this
+        leaf nor mode leaf is specified but repeat-n is set, then
+        the local AS number will be used for prepending.";
+    }
+
+    leaf mode {
+      type bgp-as-mode-type;
+      description
+        "Use a special AS number, e.g. last AS, to prepend to the
+        AS path. If neither leaf nor asn leaf is specified but
+        repeat-n is set, then the local AS number will be used for
+        prepending.";
```

Comment:
Since there is only one enum and no clear use cases for expanding this, maybe it makes sense to sense to just make this leaf `use-last-as`

Reply:
Updated the leaf to `use-last-as` as a boolean.